server/servercontext.py

```diff
@@ -132,8 +132,8 @@ async def client_connected(self, stream_reader, stream_writer):
             asyncio.CancelledError,
         ):
             pass
-        except Exception:
-            self._logger.exception()
+        except Exception as e:
+            self._logger.exception(e)
```

Comment:
Because `logger.exception` has 1 required positional argument (https://docs.python.org/3.8/library/logging.html#logging.Logger.exception) and without it will throw `TypeError`

Reply:
Hah, good catch!